lib/tasks/build-watch.js

```diff
@@ -14,12 +14,12 @@ class BuildWatchTask extends Task {
     this._runDeferred = null;
   }
 
-  run(options) {
+  async run(options) {
```

Comment:
@stefanpenner @rwjblue I think converting a public API to async function also needs the shim correct? Node will implicitly wrap the return value to a Promise.